packages/component/src/loadableEvents.js

```diff
@@ -0,0 +1,9 @@
+import EventEmitter from 'events'
```

Comment:
that is not the best package for a clientside project

Ideally make this one configurable from the outside:
- expose (to a client) a command `configureLoadableEventsController` which will accept a simple function accepting an `event`
- expose (internally) command `emitEvent` calling provided callback, or doing nothing if it was not set
- replace `loadableEvents.emit(` by `emitEvent` and worry less about complicated types or pulling in some node-related packages requiring a polyfill for browser space.

That is a general rule of interface segregation principle - do not ask (and do not provide) more than one should. Loadable needs an ability to emit a single event, let's all it a 🍌 , but you are giving it a full featured event managers - 🦍 and the rest of the 🌴 
> This is a reference to https://www.johndcook.com/blog/2011/07/19/you-wanted-banana/